dist/obs-server.spec

```diff
@@ -492,9 +538,20 @@ touch /srv/www/obs/api/last_deploy || true
 /etc/init.d/obssrcserver
 /etc/init.d/obswarden
 /etc/init.d/obsdodup
-%{_unitdir}/obsdeltastore.service
 /etc/init.d/obsservicedispatch
 /etc/init.d/obssigner
+%{_unitdir}/obsdeltastore.service
+%{_unitdir}/obsapidelayed.target
+%{_unitdir}/obsapidelayed-clockworkd.service
+%{_unitdir}/obsapidelayed-queue-consistency_check.service
+%{_unitdir}/obsapidelayed-queue-default.service
+%{_unitdir}/obsapidelayed-queue-issuetracking.service
+%{_unitdir}/obsapidelayed-queue-mailers.service
+%{_unitdir}/obsapidelayed-queue-project_log_rotate.service
+%{_unitdir}/obsapidelayed-queue-quick.service
+%{_unitdir}/obsapidelayed-queue-quick@.service
+%{_unitdir}/obsapidelayed-queue-releasetracking.service
+%{_unitdir}/obsapidelayed-sphinx.service
```

Comment:
these are frontend services, must be in obs-api package

Reply:
Moved to obs-api.